src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const shortTerm = 3;
+  const discountShortTerm = 20;
+  const longTerm = 7;
+  const discountLongTerm = 50;
```

Comment:
Static constants like these should be named in UPPER_CASE
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/63243136/487cc9e0-6a74-4eda-9c1a-e75cbd113562)